CMakeLists.txt

```diff
@@ -30,6 +30,9 @@ find_package (SeqAn3 QUIET REQUIRED HINTS lib/seqan3/build_system)
 add_subdirectory (src)
 message (STATUS "${FontBold}You can run `make` to build the application.${FontReset}")
 
+# Dependency: hclust.
+add_subdirectory (lib)
+
```

Comment:
no?

Reply:
Yes, it's much better to get rid of the git submodule because we just want to use the library instead of accessing the source code. I just pushed a commit that replaces the submodule with CMake's `FetchContent` to get the hierarchical clustering dependency. For `FetchContent` I needed to increase the CMake version to 3.11.